Source/WebCore/bindings/js/ScriptModuleLoader.cpp

```diff
@@ -508,7 +508,7 @@ void ScriptModuleLoader::notifyFinished(ModuleScriptLoader& moduleScriptLoader,
             integrity = parameters->integrity();
 
         if (!integrity.isEmpty()) {
-            if (!matchIntegrityMetadata(cachedScript, integrity)) {
+            if (!matchIntegrityMetadataAndReportHashIfNeeded(cachedScript, integrity)) {
```

Comment:
Do we need this longer name? Are there callers that don't want the _report hash_ behavior?

Reply:
I don't think there are callers that don't want to report the hash, I just thought that it's better to call out the extra functionality in the function name. Happy to revert if you think this is not necessary.

Reply:
Yes, I think the name change is not an improvement. I can see how it might be best to be explicit about this side effect, but the longer name ends up so long I don't think it adds clarity even though the short name is inaccurate. Naming remains an art, not science.

Reply:
OK, reverted!